WcaOnRails/app/models/team.rb

```diff
@@ -197,4 +197,127 @@ def acronym
   def name
     I18n.t("about.structure.#{friendly_id}.name")
   end
+
+  # rubocop:disable Metrics/CyclomaticComplexity
+  # rubocop:disable Metrics/PerceivedComplexity
+  def get_changes_of_last_month?
```

Comment:
Ruby convention: Method names only end in a question mark if they return a boolean. This method does not, so it should not end in `?`

Ruby convention: No `get_` or `set_` prefix on the method names.

Reply:
Changed.